raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -2799,4 +2779,11 @@ public void testObserverFetchWithNoLocalId() throws Exception {
     private static KafkaMetric getMetric(final Metrics metrics, final String name) {
         return metrics.metrics().get(metrics.metricName(name, "raft-metrics"));
     }
+
+    private static Optional<Integer> return0IfBoundEqualsElectionTimeout(int bound) {
```

Comment:
Tiny nit: could `OptionalInt` be returned instead?

Reply:
Yes. I removed this method in the latest commit, but used `OptionalInt` instead of `Optional<Integer>` in `MockableRandom`.

Reply:
Nice! Apologies for the drive-by comment btw., I saw your tweet about this and then my interest was piqued :)

Reply:
Suggestions for improvement are always welcome. :)